src/iwant_bot/ignore.py

```diff
@@ -0,0 +1,23 @@
+import collections
+
+
+class IgnoreList(object):
+    def __init__(self):
+        self.ignored_users = collections.defaultdict(list)
+
+    def ignore(self, user, ignored):
```

Comment:
I have a naming brainstorming result: As the class name is `IgnoreList`, adding a "X wants to ignore Y" relation looks like this:
```
ignore_list = IgnoreList()
ignore_list.ignore(X, Y)
```
It could look like
```
ignore_list.add(X, Y)
ignore_list.apply(X, IGNORES, Y)
```
and not ignoring any more
```
ignore_list.remove(X, Y)
ignore_list.apply(X, DOESNT_IGNORE, Y)
```
Like it or forget it.